src/ucs/datastruct/usage_tracker.c

```diff
@@ -190,8 +196,8 @@ static void ucs_usage_tracker_update_rank(ucs_usage_tracker_h usage_tracker)
         elem_index ++;
     )
 
-    qsort_r(elems_array, elems_count,
-            sizeof(*elems_array), ucs_usage_tracker_compare, (ucs_usage_tracker_params_t *)params);
+        qsort_r(elems_array, elems_count, sizeof(*elems_array),

```

Comment:
code style

Reply:
I guess there's a bug in clang-format

Reply:
yeah, pls format according to our code style

Reply:
seems it doesn't like the kh_foreach_key macro before this line